plugins/dup.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: dup
+spec:
+  version: v0.2.0
+  homepage: https://github.com/vash/dup
+  shortDescription: Easily and powerfully create duplicates of existing Kubernetes resources and pods that are managed by them.
```

Comment:
Try to cap this to 50 chars otherwise it won't show in fu.

Reply:
Thanks @ahmetb, made sure it follows the standar!